crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -632,6 +644,7 @@ impl Output for CliOutput {
         ]);
 
         self.print_horizontal_table(rows, header);
+        log::info!("Use -v to display full error.");
```

Comment:
```suggestion
        log::info!("An error message was truncated. Use -v to display the full error.");
```
+ only print this if at least one error was actually truncated.